scss/components/components.scss

```diff
@@ -24,11 +24,15 @@
 @import
   "stanford_basic/scss/components/brand-lockup/brand-lockup",
   "stanford_basic/scss/components/breadcrumb/breadcrumb",
-  "stanford_basic/scss/components/main-menu/main-menu",
-  "stanford_basic/scss/components/navigation/navigation",
+  // "stanford_basic/scss/components/main-menu/main-menu",
+  // "stanford_basic/scss/components/navigation/navigation",
   "stanford_basic/scss/components/pager/pager";
 
 // MY components.
 @import
+  "header/header",
```

Comment:
hmmm. Sorry about this. I will be more diligent in my merge conflict resolution. 
As an FYI, I was using the merge option `-Xtheirs` for auto resolving conflicts. I will stop that practice.